src/plugins/1/accessors/execute/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: full.html.hbs
+algolia: true
+title: execute
+---
+
+# execute
+
+{{{since "1.0.0"}}}
+
```

Comment:
Shouldn't we depreciate `execute` and document `sdk` ?

Reply:
We cannot deprecate a feature until its replacement is out of beta, which is not the case... yet.

As for the `sdk` documentation, I'm not sure how to document that yet. I wonder, shouldn't we handle that part in a dedicated issue?

Reply:
Yes I agree, this can be a different issue